doc/examples/jest+react/link.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import axe from 'axe-core';
+import { mountToDoc } from './test-helpers';
+
+import Link from './Link';
+
+test('Link has no aXe violations', (done) => {
+  const linkComponent = mountToDoc(
+    <Link page="http://www.axe-core.org">aXe website</Link>
+  );
+  const linkNode = linkComponent.getDOMNode();
+
+  axe.run(linkNode, (err, { violations }) => {
```

Comment:
We fixed that in 2.2. That's one of the reasons we build `incomplete`, so we don't have to throw abort when there's an error. We can put it in incomplete and move on.